Source/dictionaries/original.json

```diff
@@ -61,6 +75,7 @@
 		"Worst" : "Vaguely Unpleasant",
 		"Wow" : "Oh GOD This is SO Boring. Please Kill Me",
 		"WOW" : "Zzzzzzzzzzz",
+        "Years" : "Minutes",
```

Comment:
The entire file is tabulated. The best thing to do is to follow the project convention.